app/src/main/java/uk/ac/ebi/atlas/search/metadata/CellTypeWheelService.java

```diff
@@ -13,13 +14,19 @@
 @Service
 public class CellTypeWheelService {
     private final CellTypeWheelDao cellTypeWheelDao;
+    private final FeaturedSpeciesService featuredSpeciesService;
 
-    public CellTypeWheelService(CellTypeWheelDao cellTypeWheelDao) {
+    public CellTypeWheelService(CellTypeWheelDao cellTypeWheelDao,
+                                FeaturedSpeciesService featuredSpeciesService) {
         this.cellTypeWheelDao = cellTypeWheelDao;
+        this.featuredSpeciesService = featuredSpeciesService;
     }
 
     public ImmutableSet<ImmutablePair<ImmutableList<String>, String>> search(String searchTerm, String species) {
-        return cellTypeWheelDao.facetSearchCtwFields(searchTerm, species)
+        ImmutableList<String> allSpeciesNames = featuredSpeciesService.getSpeciesNamesSortedByExperimentCount();
+        var isSpeciesSearch = allSpeciesNames.contains(searchTerm);
```

Comment:
As `featuredSpeciesService.getSpeciesNamesSortedByExperimentCount()` call is returning a capitalised list of species name we also need to capitalise the `searchTerm` variable before we call `contains()` method on the list of species.

```suggestion
        var isSpeciesSearch = allSpeciesNames.contains(StringUtils.capitalize(searchTerm));
```

Reply:
Yes, and our species search term is also capitalized, so we don't need to add this utils anymore.

Reply:
Yes, for now on front end side we have this line in `LabelledSelect.js` to capitalise the species name:
```
const capitalisedName = name.charAt(0).toUpperCase() + name.slice(1)
```
But for future proofing, - as the code can change in the future - I would like you to add capitalisation in the back end side, too.
I think, it is very similar when you validate input parameters both on UI and later on back-end side as anybody can change any input parameter on frontend side with a simple tool.
It is a good practise, I believe.
Thanks